api/scpca_portal/models/sample.py

```diff
@@ -38,6 +38,25 @@ class Meta:
     def __str__(self):
         return f"Sample {self.scpca_id} of {self.project}"
 
+    @property
+    def modalities(self):
+        print(self.scpca_id)
+        modalities = []
+
+        if self.has_bulk_rna_seq:
+            modalities.append("Bulk")
+
+        if self.has_cite_seq_data:
+            modalities.append("CITE-seq")
+
+        if self.has_spatial_data:
+            modalities.append("Spatial Transcriptomics")
+
+        if len(modalities) != 0:
+            return ", ".join(modalities)
+
+        return "N/A"
```

Comment:
Not sure if that's the approach we normally use. Would setting it to N/A on the front-end for null modalities values look better?

Reply:
its kinda a mix, we have some values as N/A and others are null. I think youre right though. Null might make more sense here